package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "x-common",
+  "name": "problem-specifications",
```

Comment:
I don't believe so. As far as I can tell, this file is only used to check that canonical-data files for all exercises are compatible with the canonical-schema.